lib/earmark_parser/ast/inline.ex

```diff
@@ -255,7 +255,7 @@ defmodule Earmark.Parser.Ast.Inline do
     if match = Regex.run(@inline_ial, src) do
       [match, ial] = match
       {context1, ial_attrs} = parse_attrs(context, ial, lnb)
-      new_tags = augment_tag_with_ial(context.value, ial_attrs)
+      new_tags = augment_tag_with_ial(context.value, ial_attrs, match)
```

Comment:
Instead of taking track of the fact that we had just escaped a string I prepare for a fallback solution in the `augment_tag_with_ial` function